Gems/PhysX/Code/Source/HingeJointComponent.cpp

```diff
@@ -85,4 +88,60 @@ namespace PhysX
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
     }
+
+    // AZ::Component
+    void HingeJointComponent::Activate() {
+        JointComponent::Activate();
+        PhysX::JointInterfaceRequestBus::Handler::BusConnect(this->GetEntityId());
+    }
+
+    void HingeJointComponent::Deactivate() {
+        JointComponent::Deactivate();
+        PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
+    }
+
+    physx::PxRevoluteJoint* GetPhysXNativeRevoluteJoint(const AzPhysics::JointHandle& joint_handle,
```

Comment:
nit/standards: We don't use underscores unfortunately so variable names would be `jointHandle` and `jointSceneOwner` etc...

Reply:
I've reworked this code, this function is a protected method now, without parameter. It getter to PhysX native joint type with cache.